spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/properties/KafkaStreamsBinderConfigurationProperties.java

```diff
@@ -38,6 +38,7 @@ public KafkaStreamsBinderConfigurationProperties(KafkaProperties kafkaProperties
 	/**
 	 * Enumeration for various Serde errors.
 	 */
```

Comment:
Add `@deprecated in favor of {@link DeserializationExceptionHandler}.` or similar.